packages/p2p/src/components/my-ads/create-ad-summary.jsx

```diff
@@ -7,17 +7,41 @@ import { buy_sell } from 'Constants/buy-sell';
 import { Localize } from 'Components/i18next';
 import { useStores } from 'Stores';
 
-const CreateAdSummary = ({ offer_amount, price_rate, type }) => {
+const iff = (condition, then, other_wise) => (condition ? then : other_wise);
+
+const CreateAdSummary = ({ market_feed, offer_amount, price_rate, type }) => {
     const { general_store } = useStores();
     const { currency, local_currency_config } = general_store.client;
 
     const display_offer_amount = offer_amount ? formatMoney(currency, offer_amount, true) : '';
-    const display_price_rate = price_rate ? formatMoney(local_currency_config.currency, price_rate, true) : '';
+
+    const display_price_rate =
+        market_feed && price_rate
+            ? formatMoney(
+                  local_currency_config.currency,
+                  parseFloat(market_feed * (1 + price_rate / 100)).toFixed(2),
+                  true
+              )
+            : iff(price_rate, formatMoney(local_currency_config.currency, price_rate, true), '');
```

Comment:
Do you think `if-else` would be more readable than nesting ternary operator here? 🤔

Reply:
Yes we can. I will make the change